storageClient/api.go

```diff
@@ -46,6 +47,7 @@ type Group struct {
 	CreatedAt         time.Time `json:"created_at"`
 	LastModified      time.Time `json:"last_modified"`
 	EncryptedGroupKey string    `json:"encrypted_group_key"`
+	GroupDescription  string    `json:"group_description"`
```

Comment:
just Description for the field name as well, as for the json tag reminder to update api-spec

Reply:
will fix now!